app/Services/Auth/ApiTokenGuard.php

```diff
@@ -94,4 +94,11 @@ public function setRequest(Request $request)
 
         return $this;
     }
+
+    private function getApiToken($token)
+    {
+        return  APIToken::where('api_token', hash('sha256', $token))
+            ->where('created_at', '>', \DB::raw('DATE_SUB(NOW(), INTERVAL 1 YEAR)'))
```

Comment:
@mgonzalezs - what is token used for, and why is 1 year appropriate?  i'm wanting to understand it's effect, and how it fits with api key, user password, and signed urls.

Reply:
@jrstear if we are okay with this implementation, as discussed, please review and approve when you have time.